rest/config.go

```diff
@@ -767,9 +767,16 @@ func RunServer(config *ServerConfig) {
 
 	sc := NewServerContext(config)
 	for _, dbConfig := range config.Databases {
-		if _, err := sc.AddDatabaseFromConfig(dbConfig); err != nil {
+		c, err := sc.AddDatabaseFromConfig(dbConfig)
+		if err != nil {
 			base.LogFatal("Error opening database: %v", err)
 		}
+
+		seq, err := c.LastSequence()
```

Comment:
Calling LastSequence invokes a bucket read of _sync:seq.  We already have this value stored in the channel cache (c.changeCache.initialSequence).  Although the performance impact is negligible (as this only happens on startup), I think it would still be preferable to add a public helper method like c.InitialSequence() that retrieves the value from the cache.